### PR TITLE
disable wrong private key tests

### PR DESCRIPTION
**Pull request by matkin-msft**

### CODE REVIEW

File: tests/utils/test_crypt_util.py [hunk 1]
```diff
@@ -49,19 +50,20 @@ def test_decrypt_encrypted_text_missing_private_key(self):
         encrypted_string = load_data("wire/encrypted.enc")
         prv_key = os.path.join(self.tmp_dir, "TransportPrivate.pem")
         crypto = CryptUtil(conf.get_openssl_cmd())
-        self.assertRaises(CalledProcessError, crypto.decrypt_secret, encrypted_string, "abc" + prv_key)
-    
+        self.assertRaises(CryptError, crypto.decrypt_secret, encrypted_string, "abc" + prv_key)
+
+    @unittest.skip("Disabled due to OpenSSLBug")
```
Comment (hglkrijger) [inline tests/utils/test_crypt_util.py]: Could we rather add a predicate for Python 2.6? Or just simply remove the test if it should be disabled everywhere.

Comment (jasonzio) [reply tests/utils/test_crypt_util.py]:
```
import sys

def is_python_version_26():
    return sys.version_info.major == 2 and sys.version_info.minor == 6
```

And then, later on...

```
    @skip_if_predicate_true(is_python_version_26, "Disabled on Python 2.6")
    def test_my_bad_test(self):
        # etc etc
```

Comment (matkin-msft) [reply tests/utils/test_crypt_util.py]: it fails in any version, it just seems 2.6 is far more often.... I've made the change and we can always remove it completely if it causes issues in later versions.

---
In reply to: [197294949](https://github.com/Azure/WALinuxAgent/pull/1225#discussion_r197294949) [](ancestors = 197294949)

Comment (matkin-msft) [reply tests/utils/test_crypt_util.py]: added per Jason's comment below

---
In reply to: [197290246](https://github.com/Azure/WALinuxAgent/pull/1225#discussion_r197290246) [](ancestors = 197290246)

File: tests/utils/test_crypt_util.py [hunk 2]
```diff
@@ -34,6 +36,11 @@
 from tests.tools import *
 from subprocess import CalledProcessError
 
+
+def is_python_version_26():
+    return sys.version_info.major == 2 and sys.version_info.minor == 6
```
Comment (jasonzio) [inline tests/utils/test_crypt_util.py]: Sigh. Python 2.6 *sucks*. The .major and .minor attributes appeared in 2.7. This ought to work:

```
return sys.version_info[0] == 2 and sys.version_info[1] == 6
```

**Comment by jasonzio** [comment]
LGTM

**Pull request merged**